pkgs/native_assets_builder/test_data/drop_dylib_link/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: drop_dylib_link
```

Comment:
Please also add another test project `add_dylib_link` (or similar name) to have tests in the Dart and Flutter SDK that add new assets during the link hook to ensure we pass the assets-list from after the link hook instead of the assets from after the build hook.